examples/apps/meshdemo/frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:9.11.1
```

Comment:
Odd number node releases are unsupported after 6 months. Any reason why this isn't 12?

Reply:
I wrote the front end over 1.5 years ago, have not really changed it much since then.  I will update.